spec/intl.html

```diff
@@ -73,6 +72,30 @@ <h1>
         <p>This definition supersedes the definition provided in <emu-xref href="#sec-canonicalizetimezonename"></emu-xref>.</p>
       </ins>
     </emu-clause>
+
+    <ins class="block">
+      <emu-clause id="sup-availabletimezones" type="implementation-defined abstract operation">
+        <h1>
+          AvailableTimeZones (
```

Comment:
Do we want to land this commit ahead of intl-enumeration changing to match?

Reply:
Probably not :frowning_face:

Reply:
Intl-enumeration is not going to change to match, so unfortunately I think the best thing to do here is to keep this as it is, but later try to get it into ECMA-402 via an editorial PR once intl-enumeration reaches stage 4.